tests/sqs-emitter.js

```
@@ -0,0 +1,958 @@
+
+	});
+
+	describe('publishEvents', () => {
+
+		afterEach(() => {
+			sinon.restore();
+		});
+
+		afterEach(() => {
+			ParameterStore.clearCache();
+		});
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)